### PR TITLE
Fix indexer to work with ES 8

### PR DESCRIPTION
**Pull request by miiu96**
- Indexer v1.2.28

### CODE REVIEW

**Review by gabi-vuls — APPROVED** [review]
System test passed. 
Errors and warnings are safe. 


@@                            Log scanner                                  @@
##                            fix-indexer-to-work-with-es-8                ##
================================================================================
- Known Warnings              18
- New Warnings                2
- Known Errors                0
- New Errors                  1
- Panics                      0
================================================================================

**Review by popenta — APPROVED** [review]
@@                            Log scanner                                  @@
##                            fix-indexer-to-work-with-es-8                ##
================================================================================
- Known Warnings              20
- New Warnings                6
- Known Errors                1
- New Errors                  1
- Panics                      0
================================================================================

File: vm/systemSmartContracts/esdt.go [hunk 1]
```diff
@@ -770,6 +771,16 @@ func (e *esdt) upgradeProperties(token *ESDTDataV2, args [][]byte, isCreate bool
 		}
 	}
 
+	topics := make([][]byte, 0)
+	topics = append(topics, tokenIdentifier, big.NewInt(0).Bytes())
```
Comment (iulianpascalau) [inline vm/systemSmartContracts/esdt.go:775]: why hardcoded to 0? Is this the number of decimals? Can we have this in a local variable?

Comment (miiu96) [reply vm/systemSmartContracts/esdt.go]: topics[1] --- is the `nonce` in order to have similar logs  like the ones generated by the SFTs, NFTs and Meta ESDTs

**Review by iulianpascalau — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
System test passed.

**Pull request merged**